paper/paper.md

```diff
@@ -0,0 +1,38 @@
+
+---
+title: "covidregionaldata: An R Package for subnational and national level COVID-19 data"
+authors:
+  - name: Joseph PG. Palmer
+    affiliation: 1
+affiliations:
+ - name: Royal Holloway University of London
+   index: 1
+date: "27 April 2021"
+bibliography: paper.bib
+tags:
+  - R
+  - COVID-19
+
+output: articles::joss_article
+csl: apa.csl
+journal: JOSS
+link-citations: yes
+---
+
+# Summary 
+
+`covidregionaldata` is an R [@Rdev:2013] package to interface subnational and national level COVID-19 data, providing cleaned and checked COVID-19 case and, where available, deaths, recoveries and hospitalisation data for all countries supported. All package code is archived on Zenodo [@covidregionaldata2020] and [Github](https://github.com/epiforecasts/covidregionaldata) Currently `covidregionaldata` provides subnational data collated by official government bodies for 14 countries including the UK, India, USA, and Brazil. In addition, national level data is provided from the World Health Organisation (WHO) and European Centre for Disease Prevention and Control (ECDC). In addition, we also provide an interface to data curated by John Hopkins University (JHU), used for their [2019 Novel Coronavirus Visual Dashboard](https://github.com/CSSEGISandData/COVID-19), and Google's [covid-19-open-data](https://github.com/GoogleCloudPlatform/covid-19-open-data). The package reduces the burden of having to process data from different sources, allowing researchers to focus their efforts on research. 
```

Comment:
"by official government bodies" - I think that Canada is crowdsourced, albeit from different provincial and/or territorial government sources. I think we label at least one other (India? Brazil?) as being curated by someone who I think is non-governmental, and the U.S. data is NYTimes. Perhaps a different qualifier - "official government bodies or by credible non-governmental efforts"?